src/db/spanner/support.rs

```diff
@@ -26,10 +26,47 @@ use crate::{
 
 use super::{models::Result, pool::Conn};
 
-pub fn as_value(string_value: String) -> Value {
-    let mut value = Value::new();
-    value.set_string_value(string_value);
-    value
+pub trait ToSpannerValue {
+    fn to_spanner_value(&self) -> Value;
+}
+
+impl ToSpannerValue for String {
+    fn to_spanner_value(&self) -> Value {
+        let mut value = Value::new();
+        value.set_string_value(self.clone());
+        value
+    }
+}
+
+impl ToSpannerValue for i32 {
```

Comment:
What I really wanted to do here was:
```rust
trait ToSpannerValue for T
where T: Into<f64> {
    ...
}
```
but I ran into an error that read "error: conflicting implementations for trait ToSpannerValue", since it's possible that someone upstream could implement the `Into<f64>` trait for `String`, which would make for an ambiguous choice between the `impl ToSpannerValue for String` and `impl ToSpannerValue for Into<f64>` traits for `String`.